2-mandatory/1-writers.js

```diff
@@ -61,6 +61,17 @@ Exercise 1:
 */
 function logAllWriters() {
   // write your code to log all writers here
+for (let item of writers){
+
+let firstName = item[0];
```

Comment:
Why `item[0]`? I think you want to access the property of the object here so you can use the property name here (e.g. `item.firstName`)